webapp/channels/src/packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -464,32 +464,43 @@ export function fetchChannelsAndMembers(teamId: string): ActionFuncAsync<{channe
     };
 }
 
-export function fetchAllMyTeamsChannelsAndChannelMembersREST(): ActionFuncAsync {
+export function fetchAllMyChannelMembers(): ActionFuncAsync {
     return async (dispatch, getState) => {
         const state = getState();
         const {currentUserId} = state.entities.users;
-        let channels;
+
         let channelsMembers: ChannelMembership[] = [];
         let allMembers = true;
         let page = 0;
-        do {
-            try {
+        try {
+            while (allMembers) {
+                // Expected to disable since we don't have number of pages
```

Comment:
It's for the `eslint-ignore` since we're not supposed to loop awaits. But we have to because we don't know how many pages we have to load so we can't `Promise.all`. I'll try to clarify in the code.